### PR TITLE
Pd 576 supported kernels

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

**Comment by jrivera-px** [comment]
I generated the list from the pre-compiled KO’s archived within each release.   I don't believe we have ran on all these.   But ko should insert.

**Comment by vilasd** [comment]
@jrivera-px we can't declare its supported just looking at insert is successful

File: content/reference/knowledge-base/supported-kernels.md [hunk 1]
```diff
@@ -26,35 +26,102 @@ apt install linux-headers-$(uname -r)
...
+| **Linux Distro**	| **Kernel Version** |
+| --- | --- |
+| Ubuntu 16.04	| 4.4.0-116-generic #140-Ubuntu |
+| Fedora 27	| 4.18.18-100.fc27.x86_64 |
+| Fedora 28	| 4.18.10-200.fc28.x86_64 |
+| RHEL 7.5	| 3.10.0-1127.el7.x86_64 |
+| RHEL 7.8	| 3.10.0-1127.el7.x86_64 |
+| Debian 9	| 4.9.0-12-amd64 #1 SMP Debian 4.9.210-1 |
+
+| **Cloud BVT Distro** | **Kernel Version** |
```
Comment (vilasd) [inline content/reference/knowledge-base/supported-kernels.md:42]: Cloud Distro not BVT distros

File: content/reference/knowledge-base/supported-kernels.md [hunk 2]
```diff
@@ -26,35 +26,102 @@ apt install linux-headers-$(uname -r)
...
-1.7.1|4.19.0-041900-generic
-1.7.0|4.19.0-041900-generic
-1.6.1.4|4.19.0-041900-generic
-1.6.1.3|4.19.0-041900-generic
-1.6.1.2|4.18.16-041816-generic
-1.6.1.1|4.18.12-041812-generic
-1.6.1|4.18.11-041811-generic
-1.6.0|4.18.8-041808-generic
-1.5.1|4.18.7-041807-generic
+## Supported distros and kernel versions
```
Comment (nathan-px) [inline content/reference/knowledge-base/supported-kernels.md:29]:
```suggestion
## Qualified distros and kernel versions
```

**Pull request merged**